### PR TITLE
VULCAN-91 Wordsmithing readme

### PR DESCRIPTION
**Pull request by nhpatt**

### CODE REVIEW

**Comment by nhpatt** [comment]
👍

File: README.markdown [hunk 1]
```diff
@@ -15,61 +15,61 @@
 
 # Vulcan Architect
```
Comment (JorgeFerrer) [inline README.markdown]: Since you are at it, should we change Vulcan Architect to API Architect everywhere in this README?

Comment (nhpatt) [reply README.markdown]: I would wait until deciding a final name :P :)

Comment (JorgeFerrer) [reply README.markdown]: 👍

**Pull request merged**